lib/passport-configurator.js

```diff
@@ -307,232 +307,231 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
                   refreshToken: refreshToken,
-                }, options, done);
+                }, options, done );
             } else {
-              done(g.f('No user is logged in'));
+              done( g.f( 'No user is logged in' ) );
             }
           } else {
-            self.userIdentityModel.login(provider, authScheme, profile,
-              {accessToken: accessToken, refreshToken: refreshToken},
-              options, loginCallback(req, done));
+            self.userIdentityModel.login( provider, authScheme, profile,
+              { accessToken: accessToken, refreshToken: refreshToken },
+              options, loginCallback( req, done ) );
           }
         }
       );
   }
 
-  passport.use(name, strategy);
+  passport.use( name, strategy );
 
-  var defaultCallback = function(req, res, next) {
+  var defaultCallback = function( req, res, next ) {
     // The default callback
-    passport.authenticate(name, _.defaults({session: session},
-      options.authOptions), function(err, user, info) {
-        if (err) {
-          return next(err);
+    passport.authenticate( name, _.defaults( { session: session },
+      options.authOptions ), function( err, user, info ) {
+      if ( err ) {
+        return next( err );
+      }
+      if ( !user ) {
+        if ( !!options.json ) {
+          return res.status( 401 ).json( g.f( 'authentication error' ) );
         }
-        if (!user) {
-          if (!!options.json) {
-            return res.status(401).json(g.f('authentication error'));
-          }
-          if (options.failureQueryString && info) {
-            return res.redirect(appendErrorToQueryString(failureRedirect, info));
-          }
-          return res.redirect(failureRedirect);
+        if ( options.failureQueryString && info ) {
+          return res.redirect( appendErrorToQueryString( failureRedirect, info ) );
         }
-        if (session) {
-          req.logIn(user, function(err) {
-            if (err) {
-              return next(err);
-            }
-            if (info && info.accessToken) {
-              if (!!options.json) {
-                return res.json({
-                  'access_token': info.accessToken.id,
-                  userId: user.id,
-                });
-              } else {
-                res.cookie('access_token', info.accessToken.id,
-                  {
-                    signed: req.signedCookies ? true : false,
-                  // maxAge is in ms
-                    maxAge: 1000 * info.accessToken.ttl,
-                    domain: (options.domain) ? options.domain : null,
-                  });
-                res.cookie('userId', user.id.toString(), {
+        return res.redirect( failureRedirect );
+      }
+      if ( session ) {
+        req.logIn( user, function( err ) {
+          if ( err ) {
+            return next( err );
+          }
+          if ( info && info.accessToken ) {
+            if ( !!options.json ) {
```

Comment:
this is what I'm seeing back from login now, the only difference from what it was before is `access_token` is now `id` which is confusing. I think it would be better if we switched it to match.
```json
{
  "id": "zTAe9Rqz9a6O9kOkmEfmvi27VONHhPHH0y8cLcY3XmoYoMbEbM9uxU5xUN4CS0aD",
  "ttl": 1209600,
  "created": "2016-10-06T00:39:40.514Z",
  "userId": 1
}
```

So everything would be the same just change `id` to be `access_token`, if we do that then you could potentially consider this a feature change instead of a breaking one.